source/NVDAObjects/IAccessible/winword.py

```diff
@@ -369,6 +377,36 @@ def script_previousParagraph(self,gesture):
 		self._caretScriptPostMovedHelper(textInfos.UNIT_PARAGRAPH,gesture,None)
 	script_previousParagraph.resumeSayAllMode = sayAll.CURSOR.CARET
 
+	@script(
+		gestures=(
+			"kb:control+v",
+			"kb:control+x",
+			"kb:control+y",
+			"kb:control+z",
+			"kb:alt+backspace",
+			"kb:backspace",
+			"kb:control+backspace",
+		)
+	)
+	def script_updateBrailleAndReviewPosition(self, gesture: inputCore.InputGesture) -> None:
+		"""Helper script to update braille and review position.
+		Caret event is not always fired when control+v, control+x, control+y
+		or control+z (alt+backspace) is pressed.
+		When backspace or control+backspace is pressed and hold down, braille
+		may not always be updated. Allowing braille and review position updates
+		in L{event_caret} should fix that problem.
+		"""
+		# Ensuring braille and review position updates are allowed in caret event.
+		self._fromUpdateBrailleAndReviewPosition = True
+		# Speech output when backspace or control+backspace is pressed.
+		if gesture._get_displayName() == "backspace":
+			self.script_caret_backspaceCharacter(gesture)
+		elif gesture._get_displayName() == "ctrl+backspace":
+			self.script_caret_backspaceWord(gesture)

```

Comment:
Shouldn't this just be?

```suggestion
		if gesture.displayName == "backspace":
			self.script_caret_backspaceCharacter(gesture)
		elif gesture.displayName == "ctrl+backspace":
			self.script_caret_backspaceWord(gesture)
```